Gruntfile.js

```diff
@@ -150,23 +138,19 @@ module.exports = function (grunt) {
             src: ['omero_figure/static/figure/3rdparty/**',
                   'omero_figure/static/figure/css/**',
                   'omero_figure/static/figure/images/**',
-                  'omero_figure/static/figure/js/templates.js'], dest: 'demo/'},
+                  'omero_figure/static/figure/templates.js'], dest: 'demo/'},
         ]
       },
     },
   })
 
   grunt.loadNpmTasks('grunt-contrib-jshint')
-  grunt.loadNpmTasks('grunt-contrib-jasmine')
   grunt.loadNpmTasks('grunt-contrib-jst');
   grunt.loadNpmTasks('grunt-contrib-watch');
   grunt.loadNpmTasks('grunt-contrib-concat');
   grunt.loadNpmTasks('grunt-replace');
   grunt.loadNpmTasks('grunt-contrib-copy');
 
```

Comment:
why jshint is removed from the ``registerTask" and kept in the load.
Don't you want to use it to check the quality of the javascript code?